hack/build-and-push.sh

```diff
@@ -42,7 +43,6 @@ tkn_bundle_push() {
 function save_ref() {
     local output
     output="$(< /dev/stdin)"
-    echo "${output}"
```

Comment:
Any particular reason to remove these statements? They seem helpful for debugging purposes.

Reply:
No, I must have removed it while modifying the script. I added it back.

Reply:
I actually am keeping that one removed. It only prints the file location which is noise. The tasks being added is more helpful which is printed here. https://github.com/konflux-ci/build-definitions/pull/1167/files#diff-78a47706e6b9aca3579f93bc34adac628f9c102de7257a2fe14939130786a807R126